recipe/laravel.php

```diff
@@ -17,6 +17,13 @@
 set('writable_dirs', ['storage', 'vendor']);
 
 /**
+ * Migrate database
+ */
+task('deploy:migrate', function () {
+    run("cd {{release_path}} && ./artisan migrate --force");
```

Comment:
Does laravel support running with "no interaction". If not, we need to do something about that. :)